core/runtime/src/main/java/io/quarkus/runtime/logging/LogConfig.java

```diff
@@ -38,17 +28,44 @@
     public Level minLevel;
 
     /**
-     * Console logging config
+     * Logging categories.
+     * <p>
+     * Logging is done on a per-category basis. Each category can be independently configured.
+     * A configuration which applies to a category will also apply to all sub-categories of that category,
```

Comment:
I don't really like `A configuration` and would personally replace it with `Configuration`, but that's too minor :P

Reply:
Yeah, I just took it from the guide. For now, it's not even displayed.

Reply:
Cool